hcl2template/types.build.post-processor.go

```diff
@@ -73,7 +73,7 @@ func (cfg *PackerConfig) startPostProcessor(source SourceBlock, pp *PostProcesso
 	}
 	flatProvisinerCfg, moreDiags := decodeHCL2Spec(pp.Rest, ectx, postProcessor)
 	diags = append(diags, moreDiags...)
-	err = postProcessor.Configure(source.builderVariables(), flatProvisinerCfg, generatedVars)
+	err = postProcessor.Configure(source.builderVariables(), flatProvisinerCfg, nil)
```

Comment:
```suggestion
	err = postProcessor.Configure(source.builderVariables(), flatProvisinerCfg)
```

Reply:
I wasn't aware that this is possible 😮  thx

Reply:
var args :D